orm/src/test/java/org/hibernate/search/test/errorhandling/ErrorHandlerGivenAsInstanceTest.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.test.errorhandling;
+
+import org.hibernate.cfg.Configuration;
+import org.hibernate.search.cfg.Environment;
+import org.hibernate.search.testsupport.TestForIssue;
+
+/**
+ * Test for making sure error handlers can also be passed as instances rather than via their FQN.
+ *
+ * @author Gunnar Morling
+ */
+@TestForIssue(jiraKey = "HSEARCH-1624")
+public class ErrorHandlerGivenAsInstanceTest extends ErrorHandlingDuringDocumentCreationTest {
```

Comment:
I am not a big fan of these inheritance based tests. Hard to see what and how you tests things in this class and changes to `ErrorHandlingDuringDocumentCreationTest` might have side effects here. 

Also, from a testing POV, would it not be enough to just configure the error handler as you do here and then just boostrap and check that `SearchIntegrator#getErrorHandler` returns the right instance? A much more direct test imo.